libsol/message.c

```diff
@@ -22,10 +23,24 @@ int process_message_body(const uint8_t *message_body,
     BAIL_IF(header->instructions_length == 0);
     BAIL_IF(header->instructions_length > MAX_INSTRUCTIONS);
 
+    PRINTF("num_required_signatures = %d\n", header->pubkeys_header.num_required_signatures);
+    PRINTF("num_readonly_signed_accounts = %d\n",
+           header->pubkeys_header.num_readonly_signed_accounts);
+    PRINTF("num_readonly_unsigned_accounts = %d\n",
+           header->pubkeys_header.num_readonly_unsigned_accounts);
+    PRINTF("pubkeys_length = %d\n", header->pubkeys_header.pubkeys_length);
+    for (uint8_t i = 0; i < header->pubkeys_header.pubkeys_length; ++i) {
+        PRINTF("pubkeys[%d] = %.*H\n", i, PUBKEY_SIZE, header->pubkeys[i].data);
```

Comment:
Is there a way to prevent the loop when PRINTF is not used (DEBUG=0)?

Reply:
Compiler will remove it on its own if no instructions are inside